implementation.md

```diff
@@ -45,6 +45,8 @@ For more detailed examples, please see this [Git
 repository](https://github.com/mptcp-apps/mptcp-hello/). Do not hesitate to
 contribute here and there!
 
+### Examples on Linux
```

Comment:
Above, it is mentioned:

> This guide is for **application developers** working to add native MPTCP support when running on **Linux**.

iOS is quite different, and what is explained above is a bit particular.

I wonder if it would not be better with a new dedicated page: `Implementation Guide (iOS)`. There, we could mention that it is only for iOS, not macOS (if I understood correctly?), only the client side is supported, the Path-Manager cannot be changed, but the app can influence the scheduler, `URLSession` has to be used, not via the socket API and `connectx()` (see the [FAQ](https://www.mptcp.dev/faq.html#what-are-the-supported-operating-systems)), etc.

WDYT?